package.json

```
@@ -1,12 +1,14 @@
 {
   "name": "@digital-taco/react-draft",
-  "version": "0.4.3",
+  "version": "0.5.0",
   "description": "Develop your React components in isolation without any configuration",
-  "main": "index.js",
+  "main": "launch.prod.js",
```

Does this need to change too?
```suggestion
  "main": "bin/launch.prod.js",
```